nemo_curator/download/zh_stopwords.py

```diff
@@ -761,38 +761,38 @@
         "顺着",
         "首先",
         "︿",
-        "！",
-        "＃",
-        "＄",
-        "％",
-        "＆",
-        "（",
-        "）",
-        "＊",
-        "＋",
-        "，",
-        "０",
-        "１",
-        "２",
-        "３",
-        "４",
-        "５",
-        "６",
-        "７",
-        "８",
-        "９",
-        "：",
-        "；",
-        "＜",
-        "＞",
-        "？",
-        "＠",
-        "［",
-        "］",
-        "｛",
-        "｜",
-        "｝",
-        "～",
+        "！",  # noqa: RUF001
```

Comment:
This works but also for ease in future we can just add it to `tool.ruff.lint.per-file-ignores`